src/ElectronWrapper/lib/transcriber/index.js

```diff
@@ -1,38 +1,40 @@
 /* eslint-disable no-case-declarations */
-const fs = require('fs');
-const path = require('path');
-const { app } = require('electron').remote;
-const convertToAudio = require('../convert-to-audio/index.js');
-const convertAssemblyAIToDpeJson = require('./assemblyai/assemblyai-to-dpe/index.js');
-const assemblyAiStt = require('./assemblyai/index');
-const speechmaticsSTT = require('./speechmatics/index.js');
-const convertSpeechmaticsDpe = require('./speechmatics/speechmatics-to-dpe/index.js');
-const pocketsphinxSTT = require('./pocketsphinx-stt/index.js');
-const convertPocketsphinxOutputToDpe = require('./pocketsphinx-stt/pocketsphinx-to-dpe/index.js');
-const { getDefaultStt } = require('../../../stt-settings/default-stt.js');
+const fs = require("fs");
+const path = require("path");
+const { app } = require("electron").remote;
+const convertToAudio = require("../convert-to-audio/index.js");
+const convertAssemblyAIToDpeJson = require("./assemblyai/assemblyai-to-dpe/index.js");
+const assemblyAiStt = require("./assemblyai/index");
+const speechmaticsSTT = require("./speechmatics/index.js");
+const convertSpeechmaticsDpe = require("./speechmatics/speechmatics-to-dpe/index.js");
+const pocketsphinxSTT = require("./pocketsphinx-stt/index.js");
+const deepspeechSTT =  require("./deepspeech/index.js");
+const convertPocketsphinxOutputToDpe = require("./pocketsphinx-stt/pocketsphinx-to-dpe/index.js");
+const { getDefaultStt } = require("../../../stt-settings/default-stt.js");
+const { getDeepSpeechModelPath } = require("../../../stt-settings/credentials.js");
```

Comment:
A future potential improvement would be to have the `getDeepSpeechModelPath()` inside the `deepspeech/model.js` or something like that - and the function would use `stt-settings/credentials.js` inside, without exposing the `credentials.js` directly.

Reply:
Just because the `credentials.js` seems to be a more generic file :)